README.md

```diff
@@ -757,3 +759,32 @@ tidySingleCellExperiment::pbmc_small_nested_interactions
     ## # … with 90 more rows, and abbreviated variable names ¹​receptor, ²​ligand.name,
     ## #   ³​receptor.name, ⁴​destination, ⁵​interaction.type
     ## # ℹ Use `print(n = ...)` to see more rows
+
+#  Aggregating cells 
+
+Sometimes, it is necessary to aggregate the expression data from a group of cells into a single value. For example, when comparing groups of cells across different samples. Cells within a sample are not independent of each other and treating them as such may result in over-optimistic p-values.
```

Comment:
Again replace "expression" with gene-transcript abundance, adjusting the sentence according to sound good.

From "For example, when comparing groups of cells across different samples." to "For example, when comparing groups of cells across different samples with fixed-effect models."

let's drop the final sentence. It underly a quite huge complexity.